src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -29,24 +30,61 @@ public void promptNextPlayer(){
         }
     }
 
+    public void promptPlayAgain(){
+      System.out.println("Do you want to play again? (y/n)");
+    }
+
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
 
-        while (board.getWinner() == null){
+        int xWins = 0;
+        int oWins = 0;
+        int ties = 0;
+
+        boolean playAgain = true;
+        while (playAgain){
+            while (board.getWinner() == null && !board.isTie()){
+                board.printBoard();
+                promptNextPlayer();
+                String line = keyboardScanner.nextLine();
+                String input[] = line.split(",");
+                try {
+                    board.playMove(Integer.parseInt(input[0]), Integer.parseInt(input[1]));
+                } catch (InvalidMoveException e) {
+                    System.out.println("Invalid coordinates. Try again");
+                    promptNextPlayer();
+                }
+            }
+
             board.printBoard();
-            promptNextPlayer();
+            //win condition
+            if(!board.isTie()){
```

Comment:
Comment consistency. //win condition appears outside the branch, //tie condition on L68 appears inside the branch. Minor issue.

Reply:
This comment belongs _outside_ the block because it describes the entire if block. The comment that is poorly placed is actually the comment on the else block. This is why I hate styles with same-line structures. If we had a separate line for the else then we would have an appropriate place to put the sister comment, but we don't. I will move this inside.